### PR TITLE
Merge branch `feature/compatible-php8.2` into `develop`

### PR DESCRIPTION
**Pull request by vishalkhode1**

### CODE REVIEW

File: .github/workflows/acquia_cms_ci.workflow.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
           node-version: 12.13.1
       - uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.1
+          php-version: 8.2
```
Comment (apathak18) [inline .github/workflows/acquia_cms_ci.workflow.yml:43]: would be better if CI workflow run on both PHP 8.1 and PHP 8.2

Comment (vishalkhode1) [reply .github/workflows/acquia_cms_ci.workflow.yml:43]: @apathak18 Yes, I'll create a separate ticket to run CI on PHP 7.4/8.0/8.1 as well.

File: composer.json [hunk 2]
```diff
@@ -128,13 +128,49 @@
...
+            },
+            "drupal/schemata": {
+                "3358292 - PHP 8.2 compatibility": "https://git.drupalcode.org/project/schemata/-/merge_requests/8.patch"
+            },
+            "drupal/subrequests": {
+                "3361224 - Fix Subrequests unit tests Deprecations": "https://git.drupalcode.org/project/subrequests/-/merge_requests/16.patch"
+            },
+            "drupal/token": {
+                "3358295 - PHP 8.2 compatibility": "https://git.drupalcode.org/project/token/-/merge_requests/38.patch"
+            },
```
Comment (chandan-singh7929) [inline composer.json:173]: I would suggest to use patch before we do actual release if these MR doesn't get merge till the time.

File: .github/workflows/acquia_cms_ci.workflow.yml (hunk 1 above)
Comment (rajeshreeputra) [reply .github/workflows/acquia_cms_ci.workflow.yml:43]: +1

File: composer.json (hunk 2 above)
Comment (rajeshreeputra) [reply composer.json:173]: agree with @chandan-singh7929, if we are not providing patch then add there patches in release note so customer will be aware that, they need to apply these patches.

**Review by rajeshreeputra — APPROVED** [review]
Changes looks good to me

**Comment by chandan-singh7929** [comment]
Also, one test is failing
`There was 1 failure:

1) Drupal\Tests\acquia_cms_headless\Functional\DashboardNextjsSitesTest::testSectionAdmin
Failed asserting that actual size 0 matches expected size 1.`

**Pull request merged**